components/TextInput/TextInput.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from "react";
+import styled, { FlattenSimpleInterpolation, css } from "styled-components";
+
+interface Props extends React.Attributes {
+  multiline?: boolean;
+  rows?: number;
```

Comment:
Seems `multiline` and `rows` intersects each other. `multiline=true === rows > 1`.

So how about removing `multiline` and deciding if it's multiline or not from `rows > 1`. Then the default value of `rows` should be `1`.